capture_hdr_complete_settings.py

```diff
@@ -0,0 +1,77 @@
+"""
+Capture an HDR frame with fully configured settings for each frame.
+
+In general, taking an HDR image is a lot simpler than this as the default
+settings work for most scenes. The purpose of this example is to demonstrate
+how to configure all the settings.
+"""
+
+import zivid
+import datetime
+
+
+def _main():
+    app = zivid.Application()
+
+    print("Connecting to the camera")
+    camera = app.connect_camera()
+
+    print("Configuring the camera settings")
+    settings_collection = [camera.settings for _ in range(3)]
+
+    settings_collection[0].iris = 10
+    settings_collection[0].exposure_time = datetime.timedelta(microseconds=10000)
```

Comment:
@nedrebo This is a use case for a custom
`__copy__`
and
`__deepcopy__`
implementation
https://stackoverflow.com/questions/1500718/how-to-override-the-copy-deepcopy-operations-for-a-python-object